src/frontend/src/lib/components/nfts/NftMetadataList.svelte

```diff
@@ -51,6 +51,18 @@
 			</span>
 		{/if}
 	</ListItem>
+	{#if nonNullish(nft)}
+		<ListItem>
+			<span class="text-tertiary">{$i18n.nfts.text.token_id}</span>
+			{#if nonNullish(nft?.id)}
+				{nft?.id}
```

Comment:
at this point the nft is non nullish, right? so you don't even need the skeleton

```suggestion
			{#if nonNullish(nft.id)}
				{nft.id}
```

Reply:
yes updated